localisation/00_wc_artifact_localisation.csv

```diff
@@ -136,12 +136,12 @@ ring_of_kaldorei_empire;Ring of the Kaldorei Empire;;Ring des Kaldorei-Imperiums
 ring_of_kaldorei_empire_desc;The Ring of the Kaldorei Empire.;;Der Ring des Kaldorei-Imperiums.;;;;;;;;;;;x
 ring_of_trollbanes;Trollbane Sigil Ring;;Siegelring der Trollbanes;;;;;;;;;;;x
 ring_of_trollbanes_desc;This golden ring decorated with precious stones is a direct proof of the incredible wealth of the Trollbane family, which they have acquired over many centuries by owning a multitude of mines in the Arathi highlands.;;Dieser mit Edelsteinen verzierte goldene Ring ist ein direkter Beweis f�r den unglaublichen Reichtum der Trollbane-Familie, den sie sich �ber viele Jahrhunderte hinweg durch den Besitz einer Vielzahl von Minen im Arathi-Hochland erworben haben.;;;;;;;;;;;x
-runeblade_blood_3;Blood Runeblade;;Runenklinge des Bluts;;;;;;;;;;;x
-runeblade_blood_3_desc;Blood Runeblade.;;Runenklinge des Bluts.;;;;;;;;;;;x
-runeblade_frost_3;Frost Runeblade;;Runenklinge des Frosts;;;;;;;;;;;x
-runeblade_frost_3_desc;Frost Runeblade.;;Runenklinge des Frosts.;;;;;;;;;;;x
-runeblade_unholy_3;Unholy Runeblade;;Unheilige Runenklinge;;;;;;;;;;;x
-runeblade_unholy_3_desc;Unholy Runeblade.;;Unheilige Runenklinge.;;;;;;;;;;;x
+runeblade_blood_3;Excellent Blood Runeblade;;Ausgezeichnete Runenklinge des Bluts;;;;;;;;;;;x
+runeblade_blood_3_desc;An excellent Blood Runeblade.;;Eine ausgezeichnete Runenklinge des Bluts.;;;;;;;;;;;x
```

Comment:
Shouldn't it be...
```suggestion
runeblade_blood_3_desc;An excellent blood runeblade.;;Eine ausgezeichnete Runenklinge des Bluts.;;;;;;;;;;;x
```
Since this is not a proper name.
The same question about other runeblades.

Reply:
Ok, can change that. Just wanted to add a little more text to the description, since original creator didn't :)